joomla/joomla-cms
# [4.0] Remove obsolete import

### Summary of Changes

Removes an import which isn't needed for a very long time.

### Testing Instructions

Temporarily set text filtering to `No HTML` for Super Users in Global Configuration -> Text Filters.
Create a Custom HTML module with some HTML content.
Save and check that HTML is stripped out after saving.

### Expected result

Works like before.

### Documentation Changes Required

No.
